.github/workflows/publish.yml

```diff
@@ -73,3 +89,8 @@ jobs:
           release_name: Release ${{ steps.get-version.outputs.version }}
           body: |
             See https://github.com/simple-icons/simple-icons/releases/tag/${{ steps.get-version.outputs.version }}
+      - uses: shogo82148/actions-upload-release-asset@v1
+        name: Upload release assets
+        with:
+          upload_url: ${{ steps.create-release.outputs.upload_url }}
+          asset_path: font/*
```

Comment:
We could pack all files into a zip file. It would be better if it could include LICENSE and DISCLAIMER.

Reply:
Done. Also enabled dry-run publish workflow runs. You can see the content of the released ZIP at "Publish" workflow summaries: https://github.com/simple-icons/simple-icons-font/actions/runs/3141963909